server/index.js

```diff
@@ -5,6 +5,9 @@ const PORT = process.env.PORT || 3004;
 const app = express();
 const client = path.join(__dirname, '/../client/dist');
 app.use(express.static(client));
+app.get('/', (req, res) => {
+  res.send('Working Path');
+});
 app.listen(PORT, () => {
   console.log(`listening on port ${PORT}!`);
 });
```

Comment:
Everything is so close together?

Reply:
Yah esLint Tole me I had to.

Reply:
That is not what ESLint is telling me. May want to look at your config if you don't like this.